app/grants/migrations/0017_matchpledge_end_date.py

```diff
@@ -9,11 +9,19 @@ class Migration(migrations.Migration):
     dependencies = [
         ('grants', '0016_merge_20190305_1435'),
     ]
-
-    operations = [
-        migrations.AddField(
-            model_name='matchpledge',
-            name='end_date',
-            field=models.DateTimeField(default=grants.models.next_month),
-        ),
-    ]
+    try:
+        operations = [
+            migrations.AddField(
+                model_name='matchpledge',
+                name='end_date',
+                field=models.DateTimeField(default=grants.models.next_month),
+            ),
+        ]
+    except AttributeError:
+        operations = [
+            migrations.AddField(
+                model_name='matchpledge',
+                name='end_date',
+                field=models.DateTimeField(default=grants.models.match_pledge.next_month),
+            ),
+        ]
```

Comment:
I think this should be a `pass` instead of retrying the code that failed.

Reply:
@ksolo It's not exactly retrying the code that failed. The code in the except block is the correct migration code because it sets the default field to grants.models.match_pledge.next_month which is correct, versus grants.models.next_month which is no longer correct. I've tried using pass in the except block, but the test suite breaks because the end_date field for MatchPledge doesn't get created.